core/src/main/java/org/apache/accumulo/core/conf/SiteConfiguration.java

```diff
@@ -204,6 +204,12 @@ public static SiteConfiguration auto() {
 
   private SiteConfiguration(Map<String,String> config) {
     ConfigCheckUtil.validate(config.entrySet(), "site config");
+    config.forEach((prop, value) -> {
```

Comment:
You could also use a stream collector to collect all of the violations to include in a single exception, so users don't need to fix one, see another error, fix another, etc.

Reply:
updated in f32f471 and used some wording disucussed in #5888 for the excpetion message.  This code logs a warning in #5888 and throws an exception here.